src/fides/api/util/saas_util.py

```diff
@@ -27,6 +27,8 @@
 CUSTOM_PRIVACY_REQUEST_FIELDS = "custom_privacy_request_fields"
 UUID = "uuid"
 ISO_8601_DATETIME = "iso_8601_datetime"
+REPLY_TO = "reply-to"
+REPLY_TO_TOKEN = "reply-to-token"
```

Comment:
nit: these should use underscores instead of dashes to be consistent with the other placeholders